pkg/collector/reconcile/route.go

```
@@ -46,7 +46,7 @@ func Routes(ctx context.Context, params manifests.Params) error {
 
 	var desired []*routev1.Route
 	if isSupportedMode {
-		if r := collector.Routes(params); r != nil {
+		if r, _ := collector.Routes(params); r != nil {
```

we should propagate this error as well